Assembus/Assets/Scripts/MainScreen/Sidebar/HierarchyView/HierarchyItemController.cs

```diff
@@ -211,6 +212,57 @@ private void LateUpdate()
                 LayoutRebuilder.ForceRebuildLayoutImmediate(hierarchyView);
 
             doubleClickDetector.CheckForSecondClick();
+
+            // Scroll if dragging an item up/down
+            if (Dragging)
+                ScrollOnDrag();
+        }
+
+        /// <summary>
+        ///     Scroll list view if dragging items to bottom/top
+        /// </summary>
+        private void ScrollOnDrag()
+        {
+            // Scroll at bottom 5% of screen
+            if (Input.mousePosition.y < Screen.height / 20.0)
+            {
+                // Stop at top of scroll rect
+                if (scrollRect.normalizedPosition.y > 0.00001)
+                    // Scroll down
+                    scrollRect.normalizedPosition -= new Vector2(
+                        0.0f,
+                        GetScrollSpeedFromPosition(Math.Abs(Screen.height / 20.0f - Input.mousePosition.y))
+                    );
+            }
+            // Scroll at top 10% of screen
+            else if (Input.mousePosition.y > Screen.height - Screen.height / 10.0f)
```

Comment:
2 * Screen.height ?

Reply:
Used twice to get 90% of the screen height by subtracting 10% from it - alternatively could do height * 90 / 100.